Cookbook/Technical-Documents/FastlaneMatch.md

```diff
@@ -1,65 +1,74 @@
+We use [Fastlane Match](https://docs.fastlane.tools/actions/match/) to share code-signing identities across the team and CI. We have configured it to store the identities in our private git repo
 
-**TL;DR;**
-
-- Please do not use Match with `--readonly false` unless you really need that and you know what you are doing
-- Please don't use Xcode code signing capabilities ("Download Manual Profiles" or "Manage Certificates" buttons in Xcode's preferences pane) and don't change automatic code signing in the project settings. Always use Match.
-- If you see an error when running Match, make sure you specified the correct parameters. Consult with `fastlane/Matchfile` and `fastlane/Appfile` to inspect defaults
+**Caution**
 
-## Installing Certificates and Provisioning Profiles Locally
+- Use Match with `--readonly false` **only if** you need to generate/regenerate profiles. Otherwise existing builds on Hockeyapp will become invalid. By default it is run with `--readonly true`
+- **DO NOT** use Xcode code signing capabilities ("Download Manual Profiles" or "Manage Certificates" buttons in Xcode's preferences pane) and don't change automatic code signing in the project settings. Always use Match.
 
-### Day to day development
+## Download and install code signing identities on your machine
 
-First you need to add a developer account to your Xcode settings in `Preferences -> Accounts` menu. For that you should use a shared account (credentials are stored in the 1Password team vault). _Do not download or try to update any code signing documents via "Download Manual Profiles" or "Manage Certificates" buttons to avoid more unneeded certificates on the portal._
-
-To install development certificates locally, run:
+1. Add shared developer account credentials from 1Password to your Xcode settings in `Preferences -> Accounts` menu. 
+2. Run 
 
 ```shell
- $ bundle exec fastlane match development --team_id <enterprise team id>
+ $ bundle exec fastlane match {{Type}} --team_id {{Team-ID}} 
 ```
 
-On the first run you will be asked a password to decrypt the repository managed by Match. The password is stored in the teams 1Password vault. After you entered the password Match will install latest development certificates for all the app targets listed in the `fastlane/Matchfile` in `app_identifier` function (not scoped to particular lane with `for_lane` function). 
-Match checks if the certificates and profiles are valid and will prompt you if there is an issue. By default it is configured to run in readonly mode so it will not regenerate any certificates or profiles if they are not valid.
-
-If you need to build the app locally and run it on a device that is not yet added to the development portal you need to follow these steps:
+Note:- 
+* We have 2 developer teams - Enterprise and App-store 
```

Comment:
I did not want to mention app-store team there because these are handled specially as described in next section. I did not want a developer to read first section and run the command with just a couple of options

Reply:
Ok, make sense, but then I think the wording is confusing.

Maybe find a place upstream, like a paragraph used as introduction of them all, to say:

> We use two teams on our portal:
> - one "Enterprise" team we use for Day to Day developent and HockeyApp (see <link to paragraph>)
> - one "AppStore" team used for TestFlight and AppStore (see <link to paragraph>)

Then in the specific paragraphs, just reference them, e.g.:
 - remove the "We have two developer teams – Enterprise and App-store" line from here (since you'd have moved it upstream in the intro) but instead say "For day to day and HockeyApp, we use certificates and profiles from our Development team"
 - in the paragraph about TestFlight and AppStore, put a similar paragraph like "For TestFlight and AppStore, we use certificates and profiles from our App Store developer team"